### PR TITLE
add delivery slot to orders

### PR DESCRIPTION
**Pull request by artashes-khalatyan**

### CODE REVIEW

**Review by LookBehind — CHANGES_REQUESTED** [review]

File: src/Presentation/Nop.Web/Areas/Admin/Factories/ProductModelFactory.cs [hunk 1]
```diff
@@ -1761,7 +1766,7 @@ public virtual async Task<ProductOrderListModel> PrepareProductOrderListModelAsy
                     orderModel.OrderStatus = await _localizationService.GetLocalizedEnumAsync(order.OrderStatus);
                     orderModel.PaymentStatus = await _localizationService.GetLocalizedEnumAsync(order.PaymentStatus);
                     orderModel.ShippingStatus = await _localizationService.GetLocalizedEnumAsync(order.ShippingStatus);
-
+                    orderModel.Company = (await _companyService.GetCompanyByCustomerIdAsync(order.CustomerId)).Name;
```
[the remark this answers is not in the document]
Comment (artashes-khalatyan) [reply src/Presentation/Nop.Web/Areas/Admin/Factories/ProductModelFactory.cs:1769]: we should create uniqueness deliveryslot+date+companyid so in our case we shouldnt have such kind of cases

File: src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml [hunk 2]
```diff
@@ -13059,6 +13065,9 @@
   <LocaleResource Name="Admin.Orders.List.Store">
     <Value>Store</Value>
   </LocaleResource>
+  <LocaleResource Name="Admin.Orders.List.DeliverySlot">
+    <Value>DeliverySlot</Value>
```
[the remark this answers is not in the document]
Comment (artashes-khalatyan) [reply src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml:13069]: done

File: src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml [hunk 3]
```diff
@@ -12972,6 +12972,12 @@
   <LocaleResource Name="Admin.Orders.Info">
     <Value>Info</Value>
   </LocaleResource>
+  <LocaleResource Name="Admin.Orders.List.DeliverySlot">
+    <Value>Comapny</Value>
+  </LocaleResource>
+  <LocaleResource Name="Admin.Orders.List.Comapny">
+    <Value>Comapny</Value>
```
[the remark this answers is not in the document]
Comment (artashes-khalatyan) [reply src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml:12979]: done

File: src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml [hunk 4]
```diff
@@ -12972,6 +12972,12 @@
   <LocaleResource Name="Admin.Orders.Info">
     <Value>Info</Value>
   </LocaleResource>
+  <LocaleResource Name="Admin.Orders.List.DeliverySlot">
+    <Value>Comapny</Value>
```
[the remark this answers is not in the document]
Comment (artashes-khalatyan) [reply src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml:12976]: done

**Pull request merged**